package.json

```diff
@@ -94,18 +94,18 @@
     "@shapeshiftoss/chain-adapters": "workspace:^",
     "@shapeshiftoss/contracts": "workspace:^",
     "@shapeshiftoss/errors": "workspace:^",
-    "@shapeshiftoss/hdwallet-coinbase": "1.59.6",
-    "@shapeshiftoss/hdwallet-core": "1.59.6",
-    "@shapeshiftoss/hdwallet-keepkey": "1.59.6",
-    "@shapeshiftoss/hdwallet-keepkey-webusb": "1.59.6",
-    "@shapeshiftoss/hdwallet-keplr": "1.59.6",
-    "@shapeshiftoss/hdwallet-ledger": "1.59.6",
-    "@shapeshiftoss/hdwallet-ledger-webusb": "1.59.6",
-    "@shapeshiftoss/hdwallet-metamask-multichain": "1.59.6",
-    "@shapeshiftoss/hdwallet-native": "1.59.6",
-    "@shapeshiftoss/hdwallet-native-vault": "1.59.6",
-    "@shapeshiftoss/hdwallet-phantom": "1.59.6",
-    "@shapeshiftoss/hdwallet-walletconnectv2": "1.59.6",
+    "@shapeshiftoss/hdwallet-coinbase": "1.59.7-alpha.0",
```

Comment:
official package version upon merge and publish of: https://github.com/shapeshift/hdwallet/pull/718